latest.yaml

```diff
@@ -1,4 +1,8 @@
 # Define the values used for the "latest" tag
+base:
```

Comment:
Can we rename `base` to something like `miniforge-cuda`? This image is not a base for the wheel CI images.

Likewise for `conda` -> `ci-conda`. I'd like to use key names that match `${IMAGE_REPO//-/_}` for clarity.